src/brogue/Monsters.c

```diff
@@ -209,6 +221,11 @@ boolean monsterIsHidden(const creature *monst, const creature *observer) {
     return false;
 }
 
+/// @brief Checks if the player knows a monster's location by any means, including physical sight,
+/// clairvoyance, telepathy, or entrancement. However, the kind may be unknown (e.g. player is
+//  hallucinating or monster is submerged).
```

Comment:
> The kind may be unknown

Hmm, this disagrees with [my understanding](https://github.com/tmewett/BrogueCE/wiki/Architecture#visibility). Probably applies when hallucinating but in other cases?

Reply:
The other case is when the monster is submerged and the player is telepathic but not submerged (doesn't matter if hallucinating or not). The player knows a monster is underwater and where it is, but doesn't know what kind it is. Hovering the monster gives the "small/large" psychic emanation message. 

Maybe I should update the comment?

Reply:
If the monster is submerged but player isn't, isn't `monsterIsHidden(monst, &player)` true, meaning canSeeMonster returns false?

Reply:
You sir are absolutely correct. I pushed an update. Hope I didn't go overboard.

Reply:
It think we determined that hallucination is the only reason for not knowing the kind. How about this instead? "A monster can be "seen" but its kind unknown due to hallucination."

Reply:
Pushed an update. Feel free to reword as you see fit.

Reply:
pushed a change, what do you think?

Reply:
LGTM!